public/src/components/forms/GroupForm.tsx

```diff
@@ -253,14 +250,14 @@ export class GroupForm extends React.Component<IGroupProps, IGroupState> {
         if (this.state.selectedStudents.length === 0) {
             errors.push("Group mush have members.");
         }
+
         if (this.state.curUser
             && (this.state.curUser.link.getStatus() === Enrollment.UserStatus.STUDENT
```

Comment:
This if should be simplified through a function that returns true if these conditions are satisfied.

Reply:
Done.